site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -65,86 +33,10 @@ export const WorkspacePage: FC = () => {
         {Boolean(checkPermissionsError) && <ErrorSummary error={checkPermissionsError} />}
       </div>
     )
-  } else if (!workspace || !permissions) {
-    return <FullScreenLoader />
-  } else if (!template) {
-    return <FullScreenLoader />
+  } else if (workspace && workspaceState.matches("ready")) {
```

Comment:
Is not something we could use the Choose one component?

Reply:
Ooh good catch!